boxIt.js

```diff
@@ -0,0 +1,99 @@
+const drawLine = function (string) {
+    let num = string.length;
+    let line = '━';
+    for (i = 0; i < num; i++){
+        string = string.concat(line)
+    }
+    return string;
+}
+
+const drawTopBorder = function (string) {
+    let num = string.length;
+    let corners = '┏┓';
```

Comment:
It would make more sense to store each corner piece in its own variable. That way we don't need to call corners.slice() everytime